core/src/test/java/io/neow3j/contract/ContractParameterTest.java

```diff
@@ -160,24 +161,18 @@ public void testArrayParamCreationFromObjects_failUnSupportedObject() {
     }
 
     @Test
-    public void testArrayParam_NullObject() {
-        exceptionRule.expect(IllegalArgumentException.class);
-        exceptionRule.expectMessage("Cannot add a null object to an array contract parameter");
-        array((Object) null);
-    }
-
-    @Test
-    public void testArrayParam_NullContractParameter() {
-        exceptionRule.expect(IllegalArgumentException.class);
-        exceptionRule.expectMessage("Cannot add a null object to an array contract parameter");
-        array((ContractParameter) null);
+    public void testArrayParam_Empty() {
+        ContractParameter param = array();
+        ContractParameter[] value = (ContractParameter[]) param.getValue();
+        assertThat(value.length, is(0));
     }
 
     @Test
-    public void testArrayParam_Empty() {
-        exceptionRule.expect(IllegalArgumentException.class);
-        exceptionRule.expectMessage("At least one parameter is required");
-        array();
+    public void testArrayParam_Null() {
+        ContractParameter param = array((Object) null);
+        ContractParameter[] value = (ContractParameter[]) param.getValue();
+        assertThat(value.length, is(1));
```

Comment:
hmmm, I don't get it... why is this `1`?

A reference to a `null` Object is also in the array/stack?

Reply:
I was also confused first. But `array((Object) null)` means that we'll actually have an array with one element and that element is of parameter type `ANY` with value `null`.

Reply:
hmmmmmm... not that intuitive, to be honest.

Can we document this somewhere? Maybe in a section called "types" (or similar) in neow3j.io website?

Reply:
I think it's rather weird to do what i did in this test.
The intuitive way of getting an empty array would be `array()`, which works well. So i don't think there's a problem with the way the method works.
Anyway, I will extend the javadoc on `array(...)`.

Reply:
great!